### ISSUE TITLE
Expose metabolomics calibration information

### ISSUE THREAD

**Issue by naglepuff** [microbiomedata/nmdc-server#1456]
This is okay with me for now to unblock the release but at some point we'll need to revisit so that the calibration information can be exposed. cc @kheal

_Originally posted by @aclum in https://github.com/microbiomedata/nmdc-server/issues/1454#issuecomment-2479724044_
            
This was moved from the `workflow_execution_set`. In order to meet the scheduled release for November 2024, the simplest course of action was to remove this from the model in postgres. A follow-up task should be to re-add the information.

**Comment by aclum** [comment]
Note that the range of this spot also changed from a string to a class CalibrationInformation.

**Comment by aclum** [comment]
@kheal  the calibration identifier is now exposed in the data portal when you open the tick box for Metabolomics. Do we want to make this a link?

**Comment by aclum** [comment]
<img width="1350" height="283" alt="Image" src="https://github.com/user-attachments/assets/0fefa232-3ecd-4467-84b0-1322f5a563e0" />

**Comment by kheal** [comment]
Improved issue here: https://github.com/microbiomedata/nmdc-server/issues/1784; Adding to Data Portal board and closing this as not planned.
